runtime/near-vm-logic/src/tests/alt_bn128.rs

```diff
@@ -0,0 +1,222 @@
+use super::{fixtures::get_context, vm_logic_builder::VMLogicBuilder};
+use near_vm_errors::{HostError, VMLogicError};
+use std::fmt::Write;
+
+/// Converts a sequence of integers to a single little-endian encoded byte
+/// vector. `u8` literals like `0u8` are treated as u8, hex literals like
+/// `0xbeef` are treated as `u256`.
+///
+/// Comas (`,`) can be used for readability, but don't affect semantics
+macro_rules! le_bytes {
+    ($($lit:tt)*) => (
+        {
+            #[allow(unused_mut)]
+            let mut buf: Vec<u8> = Vec::new();
+            $(parse_le_bytes(stringify!($lit), &mut buf);)*
+            buf
+        }
+    )
+}
+
+fn parse_le_bytes(s: &str, buf: &mut Vec<u8>) {
+    if s == "," {
+        return;
+    }
+    if let Some(byte) = s.strip_suffix("u8") {
+        buf.push(byte.parse::<u8>().unwrap());
+        return;
+    }
+    assert!(s.starts_with("0x"));
+    let s = &s[2..];
```

Comment:
If we had `let-else` I'd write this as:

```suggestion
    let Some(s) = s.strip_prefix("0x") else { panic!("does not start with 0x") };
```

but 

```suggestion
    let s = s.strip_prefix("0x").expect("0x prefix was expected");
```

would also work